docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -286,254 +286,568 @@ visitor_rate_ecs::
 * Models occurrences of processes in paths.
 * Detects activity in unusual paths (using the {ml-docs}/ml-rare-functions.html#ml-rare[`rare` function]).
 
-Activities in unusual paths may indicate execution of malware or persistence 
-mechanisms. Windows payloads often execute from user profile paths.
+Required {beats}:::
+
+* {winlogbeat} (Windows)
+
+Required ECS fields when not using {beats}:::
+
+* `host.name`
+* `process.name`
+* `user.name`
+* `process.working_directory`
+* `event.action`
+* `agent.type`
 
 windows_anomalous_process_creation::
 
-* For host activity logs where `agent.type` is `winlogbeat`.
+Identifies unusual parent-child process relationships that can indicate
+malware execution or persistence mechanisms.
++
+Malicious scripts often call on other applications and processes as part of
+their exploit payload. For example, when a malicious Office document runs
+scripts as part of an exploit payload, Excel or Word may start a script
+interpreter process, which, in turn, runs a script that downloads and executes
+malware. Another common scenario is Outlook running an unusual process when
+malware is downloaded in an email.
++
+Monitoring and identifying anomalous process relationships is an excellent way
+of detecting new and emerging malware that is not yet recognized by anti-virus
+scanners.
+
+Job details:::
+
+* Analyzes host activity logs where `agent.type` is `winlogbeat`.
 * Models occurrences of process creation activities (`partition_field_name` is 
   `process.parent.name`).
 * Detects process relationships that are rare compared to other process 
   relationships (using the {ml-docs}/ml-rare-functions.html#ml-rare[`rare` function]).
 
-Looks for unusual process relationships which may indicate execution of malware 
-or persistence mechanisms.
+Required {beats}:::
+
+* {winlogbeat} (Windows)
+
+Required ECS fields when not using {beats}:::
+
+* `host.name`
+* `process.name`
+* `user.name`
+* `process.parent.name`
+* `event.action`
+* `agent.type`
 
 windows_anomalous_script::
 
-* For host activity logs where `agent.type` is `winlogbeat`.
-* Models occurrences of powershell script activities.
-* Detects unusual powershell script execution compared to other powershell 
+Searches for PowerShell scripts with unusual data characteristics, such as
+obfuscation, that may be a characteristic of malicious PowerShell script text
+blocks.
+
+Job details:::
+
+* Analyzes host activity logs where `agent.type` is `winlogbeat`.
+* Models occurrences of PowerShell script activities.
+* Detects unusual PowerShell script execution compared to other PowerShell 
   script activities (using the 
   {ml-docs}/ml-info-functions.html#ml-info-content[`high_info_content` function]).
 
-Looks for unusual powershell scripts that may indicate execution of malware, or 
-persistence mechanisms.
+Required {beats}:::
+
+* {winlogbeat} (Windows)
+
++
+NOTE: You can only run this job when Winlogbeat is used to ship data.
```

Comment:
Attempting to clarify whether there is a failure to run (i.e. job is visible but fails) or whether you never see this job:
```suggestion
NOTE: This job is available only when you use {winlogbeat} to ship data.
```

Reply:
Yep - will update all the relevant notes.